tasks/broad/ImputationTasks.wdl

```diff
@@ -19,7 +19,6 @@ task CalculateChromosomeLength {
     disks: "local-disk ${disk_size_gb} HDD"
     memory: "${memory_mb} MiB"
     cpu: cpu
-    maxRetries: 2
```

Comment:
remind me why we removed maxRetries? my understanding is this is helpful when practical (i.e. not on a huge task) to not fail on transient GCP/docker/whatever hiccups. but if so we might want to add it back later so that for testing we can identify those?